src/lib/bind/accessors.html

```diff
@@ -48,7 +48,10 @@
 
       _propertySet: function(property, value, effects) {
         var old = this.__data__[property];
-        if (old !== value) {
+        // NaN is always not equal to itself,
+        // if old and value are both NaN we treat them as equal
+        // x !== x is 10x faster, and equivalent to isNaN(x)
+        if (old !== value && !(old !== old && value !== value)) {
```

Comment:
Seems like `(old !== value && old==old && value==value)` is slightly more readable.